cmd/workspace.go

```diff
@@ -198,6 +203,85 @@ func newWorkspaceUserListCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
+/*
+	Workspace teams commands
+*/
+
+func newWorkspaceTeamRootCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "team",
+		Short: "Manage Workspace Team resources",
+		Long:  "Teams can be added or removed from Workspaces",
+	}
+	cmd.AddCommand(
+		newWorkspaceTeamAddCmd(out),
+		newWorkspaceTeamUpdateCmd(out),
+		newWorkspaceTeamRmCmd(out),
+		newWorkspaceTeamListCmd(out),
+	)
+	return cmd
+}
+
+func newWorkspaceTeamAddCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "add TEAM",
+		Short:   "Add a Team to a Workspace",
+		Long:    "Add a Team to a Workspace",
+		Example: "astro workspace team add <team-id> --workspace-id <workspace-id> --role <role>",
+		Args:    cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return workspaceTeamAdd(cmd, out, args)
+		},
+	}
+	cmd.PersistentFlags().StringVar(&workspaceID, "workspace-id", "", "workspace to add the team to")
+	cmd.PersistentFlags().StringVar(&workspaceRole, "role", houston.WorkspaceViewerRole, "workspace role assigned to team")
+	return cmd
+}
+
+func newWorkspaceTeamUpdateCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update team role",
```

Comment:
Shouldn't this command be under the following structure ?
```bash
astro workspace update TEAM-ID --role=new_role
```

You use a flag below `--role` for users to update the team (which is good, if we want to add more fields in the future we can easily add more flags), but here in the `Use` property you specify that the user has to pass the role as an argument. Morever, you specify Args to be exactArgs(1)

Team ID should remain as an argument though.

Reply:
done